engine/src/flutter/impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -914,10 +914,10 @@ KernelSamples GenerateBlurInfo(BlurParameters parameters) {
   result.sample_count =
       ((2 * parameters.blur_radius) / parameters.step_size) + 1;
 
-  // Chop off the last samples if the radius >= 3 where they account for < 1.56%
-  // of the result.
+  // Chop off the last samples if the radius >= 16 where they account for
+  // < 1.56% of the result.
```

Comment:
Is it still 1.56% if you increase the radius size? I would think that the contribution %age would go way down at that radius.

Reply:
I didn't measure it, that's from the article on the lerp hack.  I think it's probably still a good rule of thumb, it obviously isn't going to be exact for every radius.

Reply:
I updated the comment to reflect it isn't exactly 1.56%